### PR TITLE
Added doc changes for logging_global

### PR DESCRIPTION
**Pull request by roverflow**
##### SUMMARY
This PR fixes - #1072 

Update documentation - ios_logging_global for severity attribute within buffered.

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

File: plugins/modules/ios_logging_global.py [hunk 1]
```diff
@@ -38,7 +38,9 @@
             description: Logging buffer size
             type: int
           severity: &severity
-            description: Logging severity level
+            description:
+              - Logging severity level
+              - Idempotancy is not maintained for this attribute as value not shown in running-config
```
Comment (KB-perByte) [inline plugins/modules/ios_logging_global.py:43]: maybe this behavior is specific to the version we currently test with, let's add integration tests to check this nature, and add a comment there. Incase this fails the test in the future we would know to update the doc and fix the test.

File: tests/integration/targets/ios_logging_global/tests/cli/merged.yaml [hunk 2]
```diff
@@ -57,5 +57,23 @@
...
+
+    - name: Test - [merged] merge console severity with device configuration (idempotent)
+      register: resultdeb
+      cisco.ios.ios_logging_global: *id001
+
+    - name: Test - [merged] assert that debugging value is not idempotent
+      ansible.builtin.assert:
+        that:
+          - resultdeb['changed'] == true
+          - "'logging buffered xml 5099 debugging' in resultdeb['commands']"
```
Comment (KB-perByte) [inline tests/integration/targets/ios_logging_global/tests/cli/merged.yaml:76]: assert the after dict to confirm that it impacts idempotency.

**Review by KB-perByte — CHANGES_REQUESTED** [review]

File: plugins/modules/ios_logging_global.py [hunk 3]
```diff
@@ -38,7 +38,10 @@
             description: Logging buffer size
             type: int
           severity: &severity
-            description: Logging severity level
+            description:
+              - Logging severity level
+              - Idempotancy is not maintained for this attribute for the value debugging
```
Comment (NilashishC) [inline plugins/modules/ios_logging_global.py:43]:
```suggestion
              - Idempotency is not maintained for this attribute for the value 'debugging'
```

File: plugins/modules/ios_logging_global.py [hunk 4]
```diff
@@ -38,7 +38,10 @@
...
+                as value not shown in running-config
```
Comment (NilashishC) [inline plugins/modules/ios_logging_global.py:44]:
```suggestion
                since this value is not shown in running-config.
```

**Review by KB-perByte — APPROVED** [review]

**Pull request merged**